package.json

```diff
@@ -2,13 +2,25 @@
   "name": "rollup-plugin-livereload",
   "version": "2.0.5",
   "description": "Rollup plugin for LiveReload that watches the bundle and reloads the page on change",
-  "main": "dist/index.cjs.js",
-  "module": "dist/index.es.js",
-  "jsnext:main": "dist/index.es.js",
-  "types": "index.d.ts",
+  "type": "module",
```

Comment:
The package type now defaults to `"module"`, which ensures that Node.js will evaluate the files as ESM.